debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/AbstractIncrementalSnapshotChangeEventSource.java

```diff
@@ -620,6 +621,9 @@ private List<String> expandDataCollectionIds(List<String> dataCollectionIds) {
                     return ids.isEmpty() ? Stream.of(x) : ids.stream();
                 })
                 .collect(Collectors.toList());
+        HashSet<String> uniqueDataCollections = new HashSet<>();
```

Comment:
@akanimesh7 Thanks for the PR. Unfortunately I am afraid that the result of this operation will be the loss of order intended to execute the snapshot. The deduplication should keep the order of tables prescribed.

Reply:
@jpechane Sorry, i missed that. Changed it to LinkedHashSet to preserve the oder.